airbyte-cdk/python/airbyte_cdk/sources/file_based/file_based_source.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import logging
+import traceback
+from abc import ABC
+from typing import Any, Dict, List, Mapping, Optional, Tuple
+
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.file_based.default_file_based_availability_strategy import DefaultFileBasedAvailabilityStrategy
```

Comment:
I don't see this file anymore. Was it deleted?

Reply:
Strange - no, it's still there. Where were you looking?

Reply:
not sure what happened.  🤷